external/mmsegmentation/segmentation_tasks/apis/segmentation/model_wrappers/blur.py

```diff
@@ -90,6 +90,6 @@ def postprocess(self, outputs: Dict[str, np.ndarray], metadata: Dict[str, Any]):
                 outputs["saliency_map"][0],
                 (metadata["original_shape"][1], metadata["original_shape"][0]),
             )
-            metadata["feature_vector"] = outputs["feature_vector"] 
+            metadata["feature_vector"] = outputs["feature_vector"].reshape(-1)
```

Comment:
Is this metadata called "feature_vector" eventually renamed to "representation_vector" when used to build a `TensorEntity` object?

Reply:
Yes. They seems to be "feature_vector" usually in raw inference, and then packaged as "representation_vector" for metadata.